data.json

```diff
@@ -1229,4 +1229,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/14886766/127783620-6b94c878-9fd2-40c5-a62d-995818f0caa3.jpeg",
     "issueNumber": 630
   }
```

Comment:
Please add a comma at the end of the curly bracket on line 1231. Then everything should be ok.